common/src/main/scala/explore/utils/ReactTableHelpers.scala

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) 2016-2020 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.utils
+
+import cats._
+import eu.timepit.refined.auto._
+import eu.timepit.refined.types.string._
+import explore._
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.util.{ Display, Enumerated }
+import lucuma.ui.forms._
+import lucuma.ui.optics._
+import monocle.Lens
+import react.semanticui.elements.button.Button
+import reactST.reactTable.mod._
+
+import scalajs.js
+
+object ReactTableHelpers {
+
+  /**
+   * Create a AgGridColumn Builder for a column which edits the value in a FormInputEv.
```

Comment:
`AgGridColumn`?

Reply:
Oops.

Reply:
Looks like I did too much copy/paste of documentation from the ag-grid implementation. I'll go through that.